gatsby-theme-intro/content/social.yaml

```diff
@@ -30,3 +30,6 @@
 
 - name: youtube
   url:
+
+- name: medium
+  url: https://medium.com/@username
```

Comment:
Just one thing: can you please leave the url value blank? This way the medium icon won't be displayed unless the URL will be explicitly set in the code.

Reply:
Should be fixed now!